wip-0018.md

```diff
@@ -25,15 +25,69 @@ format!("inner: {:?}", inner)
+```
+
+#### Created in retrieval/aggregation
+
+The other kind of `UnhandledIntercept` can be seen in data request `463bb51d32942fb48edb88684a47440d8165bc97a1912acfab220f6de0e87ff5`. In this request a majority of witnesses report an error, and the result of the request is a mode of the different errors, which happens to be an `UnhandledIntercept` error. This is the tally using the old validation rules:
+
+```
+Tally: RadonTypes::RadonError(UnhandledIntercept { inner: None, message: Some("inner: Decode { from: \"cbor::value::Value\", to: \"RadonArray\" }") })
+```
+
+And this is the tally using the logic from WIP-0018:
+
+```
+Tally: RadonTypes::RadonError(UnhandledInterceptV2 { inner: None })
+```
+
+However, in this case it is impossible to recover the original error, because the error is created in the retrieval stage and we only have the error message, which is removed from the tally result:
```

Comment:
```suggestion
However, in this case, it is impossible to recover the original error because the error is created in the retrieval stage and we only have the error message, which is removed from the tally result:
```